packages/app-project/src/helpers/fetchTranslations/fetchTranslations.spec.js

```diff
@@ -5,28 +5,23 @@ import fetchTranslations from './'
 describe('helpers > fetchTranslations', function () {
   before(function () {
     nock('https://panoptes-staging.zooniverse.org/api')
+    .persist()
     .get('/translations')
     .query(query => query.translated_type === 'project')
     .reply(200, {
       translations: [
         {
-          language: 'fr',
+          language: 'zh-cn',
           strings: {
-            display_name: 'Le project'
+            display_name: 'Chinese translation'
           }
         },
         {
-          language: 'en',
+          language: 'fr',
           strings: {
-            display_name: 'test project'
+            display_name: 'Le project'
           }
-        }
-      ]
-    })
-    .get('/translations')
-    .query(query => query.translated_type === 'project' && query.language === 'fr,en')
```

Comment:
This matcher tests against the expected request query, so you’ll want to keep it.